### PR TITLE
[BOND-8207] lib updates

### PR DESCRIPTION
**Pull request by DR9885**

### CODE REVIEW

File: src/EventHorizon.EventSourcing/Aggregates/Aggregator.cs [hunk 1]
```diff
@@ -245,8 +246,12 @@ public async Task DropAllAsync(CancellationToken ct)
     {
         await _store.DropDatabaseAsync(ct).ConfigureAwait(false);
         await _streamingClient.GetAdmin<Event>().DeleteTopicAsync(_stateType, ct: ct).ConfigureAwait(false);
-        await _streamingClient.GetAdmin<Command>().DeleteTopicAsync(_stateType, ct: ct).ConfigureAwait(false);
-        await _streamingClient.GetAdmin<Request>().DeleteTopicAsync(_stateType, ct: ct).ConfigureAwait(false);
+
+        if(_stateDetail.HandlerDict[typeof(Command)].Count != 0)
```
Comment (zblocker280) [inline src/EventHorizon.EventSourcing/Aggregates/Aggregator.cs:250]: Can you explain why these changes are necessary? Just curious what's going on.

**Review by zblocker280 — APPROVED** [review]

File: src/EventHorizon.EventSourcing/Aggregates/Aggregator.cs (hunk 1 above)
Comment (DR9885) [reply src/EventHorizon.EventSourcing/Aggregates/Aggregator.cs:250]: because if the topic didn't exist it would throw a 500 http error code.
